frontend/src/app/page.tsx

```diff
@@ -1,101 +1,53 @@
-import Image from "next/image";
+"use client";
+
 import styles from "./page.module.css";
 import { WeatherForecast } from "@/app/weather";
+import {
+  AuthenticatedTemplate,
+  MsalProvider,
+  UnauthenticatedTemplate,
+} from "@azure/msal-react";
+import { EventType } from "@azure/msal-browser";
+import { msalInstance } from "@/app/msalInstance";
 
-// @ts-ignore
-export default function Home() {
-  return (
-    <main className={styles.main}>
-      <div className={styles.description}>
-        <div>
-          Get started by editing&nbsp;
-          <code className={styles.code}>src/app/page.tsx</code>
-          <code>
-            <WeatherForecast />
-          </code>
-        </div>
-        <div>Test</div>
-        <div>
-          <a
-            href="https://vercel.com?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            By{" "}
-            <Image
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className={styles.vercelLogo}
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
-        </div>
-      </div>
-
-      <div className={styles.center}>
-        <Image
-          className={styles.logo}
-          src="/next.svg"
-          alt="Next.js Logo"
-          width={180}
-          height={37}
-          priority
-        />
-      </div>
-
-      <div className={styles.grid}>
-        <a
-          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className={styles.card}
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2>
-            Docs <span>-&gt;</span>
-          </h2>
-          <p>Find in-depth information about Next.js features and API.</p>
-        </a>
+msalInstance.initialize().then(() => {
```

Comment:
Where are the docs for this msal-implementation?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md